### ISSUE TITLE
Error with HA 2023.04

### ISSUE THREAD

**Issue by steeeeev0** [ben8p/lovelace-auto-reload-card#8]
This card was working fine for me until 2023.04. I have tried the fix mentioned in another issue, but that did not seem to help.

The error in HA is as follows:
`http://10.0.30.22:8123/hacsfiles/lovelace-auto-reload-card/auto-reload-card.js?hacstag=321140869961031:24:29 TypeError: null is not an object (evaluating 'header.classList')`

Does anyone have any hints here?

**Comment by bugraacar** [comment]
Did You Find Any Solution It Stopped Working For Me

**Comment by steeeeev0** [comment]
> Did You Find Any Solution It Stopped Working For Me

Yes, though it has nothing to do with this addin. I'm loading custom js to lovelace that forces a refresh at a timed interval. This affects all dashboards and views.

This js script that will force a reload on a timed interval (15 minutes) and saved it as `refresh.js` in the `www` folder:
```
function refreshPage() {
    console.log("Refreshing page on timed interval...")
    location.reload()
}
setInterval(refreshPage, 900000);
```

Then, in the `frontend` section of `configuration.yaml` I added an `extra_module_url`:
```
frontend:  
   themes: !include_dir_merge_named themes  
   extra_module_url:    
      - /local/refresh.js

```

**Comment by bugraacar** [comment]
Thank you,
It worked for me,
